packages/lib/src/common/utils/index.js

```diff
@@ -662,6 +662,12 @@ class Utils {
         proGalleryRole === 'application' ? 'gallery application' : 'region',
     };
   }
+
+  focusGalleryElement(element){
+    if(element) {
```

Comment:
Why do you need the `if`. On which scenario this function is called without an `element`?